mathjax3-ts/input/tex/action/ActionConfiguration.ts

```diff
@@ -0,0 +1,82 @@
+/*************************************************************
+ *
+ *  Copyright (c) 2018 The MathJax Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+
+/**
+ * @fileoverview Configuration file for the action package. Note that this is
+ *     based on AMS package and Newcommand utilities.
+ *
+ * @author v.sorge@mathjax.org (Volker Sorge)
+ */
+
+import {Configuration} from '../Configuration.js';
+import TexParser from '../TexParser.js';
+import {CommandMap} from '../SymbolMap.js';
+import {ParseMethod} from '../Types.js';
+import AmsMethods from '../ams/AmsMethods.js';
+import NewcommandUtil from '../newcommand/NewcommandUtil.js';
```

Comment:
I just noticed this dependency.  I don't see where it is used.  Is it essential?

Reply:
@zorkow, I know this one is already merged, but are these dependencies needed?  I didn't spot this until I noticed it in other packages.